src/network.cpp

```diff
@@ -4,11 +4,37 @@
  */
 #include "network.h"
 #include "config.h"
+#include "logging.h"
 #include "utility.h"
 #include <algorithm>
```

Comment:
I'm not really sure why clang format wants `network.h` first, but whatever.

Reply:
Maybe it's treated differently because it's the corresponding .h file for this .cpp file.

Reply:
I was thinking that too but didn't necessarily see the same behavior in other cpp files.